PlayList.java

```diff
@@ -35,32 +35,53 @@ public Track getTrack(int index) {
      *  If the list is full, does nothing and returns false.
      *  Otherwise, appends the track and returns true. */
     public boolean add(Track track) {
-        //// replace the following statement with your code
+        if(this.size>=this.maxSize)
+            return false;
```

Comment:
Always use `{}` to scope your conditional calculation - even when there is only one line of code.
```suggestion
        if(this.size>=this.maxSize) {
            return false;
        }
```